install-panel.sh

```diff
@@ -209,37 +266,27 @@ function detect_distro {
   OS_VER_MAJOR=$(echo "$OS_VER" | cut -d. -f1)
 }
 
-function check_os_comp {
-  if [ "$OS" == "ubuntu" ]; then
-    PHP_SOCKET="/run/php/php7.4-fpm.sock"
-    if [ "$OS_VER_MAJOR" == "18" ]; then
-      SUPPORTED=true
-    elif [ "$OS_VER_MAJOR" == "20" ]; then
-      SUPPORTED=true
-    else
-      SUPPORTED=false
-    fi
-  elif [ "$OS" == "debian" ]; then
-    PHP_SOCKET="/run/php/php7.4-fpm.sock"
-    if [ "$OS_VER_MAJOR" == "9" ]; then
-      SUPPORTED=true
-    elif [ "$OS_VER_MAJOR" == "10" ]; then
-      SUPPORTED=true
-    else
-      SUPPORTED=false
-    fi
-  elif [ "$OS" == "centos" ]; then
-    PHP_SOCKET="/var/run/php-fpm/pterodactyl.sock"
-    if [ "$OS_VER_MAJOR" == "7" ]; then
-      SUPPORTED=true
-    elif [ "$OS_VER_MAJOR" == "8" ]; then
-      SUPPORTED=true
-    else
-      SUPPORTED=false
-    fi
-  else
-    SUPPORTED=false
-  fi
+check_os_comp() {
+  SUPPORTED=false
```

Comment:
Is this redundant? The default case, handled with the wildcard, also sets `SUPPORTED=false`. I have not tested if this is the case or not.

Reply:
Yes, I thought about removing and then forgot. Thanks for reminding.